Podfile

```diff
@@ -1,5 +1,8 @@
 # Uncomment the next line to define a global platform for your project
 # platform :ios, '9.0'
+source 'https://github.com/twilio/cocoapod-specs'
+source 'https://github.com/CocoaPods/Specs.git'
```

Comment:
`source 'https://github.com/CocoaPods/Specs.git'` is enough.
 we don't need `source 'https://github.com/twilio/cocoapod-specs'`